discordbot/README.md

```diff
@@ -34,11 +34,13 @@ pip install python-dotenv
 
 1. Login into your discord account in https://discord.com/developers/applications
 2. On the "Applications" tab, select "New Application" and name it something like "GST Bot". This will create a new application.
-3. For creating a bot, we need to go into the "GST Bot" application previously created, and on the "Bot" tab select "Add Bot". Note: On this same page you can manage the bot permissions with regards to the server it will be added.
-4. Next we need to obtain the "Client ID". For that we choose the "OAuth2" tab and copy our "Client ID" under "Client".
-5. At this stage the bot is created and we are ready to invite it to one of our servers. To do so we need to access:
+3. For creating a bot, we need to go into the "GST Bot" application previously created, and on the "Bot" tab select "Add Bot".
+4. Allow Privileged Gateway Intents as shown below. <img width="1281" alt="Screen Shot 2021-11-01 at 4 19 55 PM" src="https://user-images.githubusercontent.com/18151143/139737415-29873019-7766-465c-9480-5315acd8df2e.png">

```

Comment:
Can we have "Allow privileged gateway intents by enabling MESSAGE CONTENT INTENT in tab Bot or so" instead of the picture?

Just to be more clean set of instructions

Reply:
ya